docs/en/cloud/Alluxio-Presto-AMI.md

```diff
@@ -60,7 +60,7 @@ The AMI with the longer name is derived from the shorter one but is constrained
 
 Among the large table of instance types and their specifications, scroll down to search for the `r4.4xlarge` instance type.
 
-**Warning:** `r4.4xlarge` is provisioned with the minimum amount of resources needed to run this AMI.
+> **Warning:** `r4.4xlarge` is provisioned with the minimum amount of resources needed to run this AMI.
```

Comment:
Is this needed? what it used for?

Reply:
styling. it shows up with a slight indentation